src/styles/Global.style.js

```diff
@@ -0,0 +1,18 @@
+import { injectGlobal } from 'styled-components';
+
+injectGlobal`
+  body {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+  }
+
+  .AppContainer {
```

Comment:
you still using classNames?

Reply:
I use .AppContainer in 2 places, the header and the entire app. Is there a better way to do it?

Reply:
Yes, you shouldn't use classnames, you can create an AppContainer with Styled componets that implements this. Check https://www.styled-components.com/docs/basics#getting-started

Reply:
Makes sense! Thanks!

I also have the class `mt` that is just a margin top. Do you an example of how to handle this without `css classes`?

Reply:
haven't used `mt` before, so I believe that it has a css that you can copy. (Sort of thing I did with BootStrap buttons)